layouts/shortcodes/blocks/samples-section.html

```diff
@@ -0,0 +1,11 @@
+{{ $url_text := .Get "url_text" }}
+{{ $api := .Get "api" }}
+{{ $last_updated := "" }} 
+{{ $version := .Get "kfctl" }} 
+<div class="col-lg-12 mb-5 mb-lg-0 ">
+  {{ with .Get "title" }}<h4 class="h3 mt-3">{{ . }}</h4>{{ end }}
+  {{ with getJSON $api }} {{ $last_updated = (index (index (index (index . 0) "commit") "committer") "date") | dateFormat "2006/01/02" }}
+  <p class="text-muted">{{ "Last update " }} {{ $last_updated }} {{ with $version }}{{ "Kubeflow " }}{{ . }}{{ end }}</p>{{ end }} 
+  <p class="mb-0">{{ .Inner }}</p>
+  {{ with .Get "url" }}<p><a href="{{ . }}">{{ with $url_text }}{{ $url_text }}{{ else }}{{ "Read" }}{{ end }}</a></p>{{ end }}
```

Comment:
Would it be more meaningful to change `"Read"` to `"Go to sample"` now that we know this particular block is specifically for samples?

Another suggestion: I think it's unlikely we'll ever use the `url_text` param. Should we remove it, to make the code simpler?